Description: Correct a typo in logging
 .
 securityonion-elsa-extras (20131117-1ubuntu0securityonion35) precise; urgency=low
 .
   * Update the localhost peer
Author: Scott Allen Runnels <srunnels@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- securityonion-elsa-extras-20131117.orig/bin/securityonion_elsa_register.rb
+++ securityonion-elsa-extras-20131117/bin/securityonion_elsa_register.rb
@@ -9,7 +9,7 @@ require 'optparse'
 
 module Logging
   LOG_DIR = "/var/log/nsm/so-elsa"
-  LOG_FILE = "/elsa_registration_logfile-#{Time.now.to_i}.log"
+  LOG_FILE = LOG_DIR + "/elsa_registration_logfile-#{Time.now.to_i}.log"
   unless File.directory? LOG_DIR
     Dir::mkdir LOG_DIR
   end
